sonar-project.properties

```diff
@@ -1,4 +1,4 @@
 sonar.projectKey=sakuli:sakuli
 sonar.sources=packages/sakuli-cli,packages/sakuli-core,packages/sakuli-commons,packages/sakuli-legacy
-sonar.exclusions=**/node_modules/**/*
+sonar.exclusions=**/node_modules/**/*,**/*.spec.*,**/*.test.*,**/*.config.*
```

Comment:
should we also consider adding `**/__mocks__/**/*`, `**/__tests__/**/*` and `**/__snapshots__/**/*`?